### PR TITLE
docs(publish): document publishing and deploying via CLI

### PR DESCRIPTION
**Pull request by bstrie**
## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: docs/Running/Publish.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Publishing and deploying applications via Enarx
```
Comment (platten) [inline docs/Running/Publish.md:1]: This documentation should be versioned. As we are post-release, we will need to use an override until the next upcoming release. To do this, create a directory called "override" in the root and put the documents there (as well as please commit this to the enarx repository under docs for the upcoming release).

**Review by platten — CHANGES_REQUESTED** [review]
Please move `docs/Running/Publish.md` to `override/Running/Publish.md` and create a PR to add `Running/Publish.md` under the `docs` directory in the `enarx` repository.

**Review by nickvidal — APPROVED** [review]
Looks good, but I wasn't able to test it, as I got this error message while trying to register myself: 

```
Error: Failed to save user credentials
Caused by:
    0: Platform secure storage failure: zbus error: D-Bus handshake failed: Unexpected server AUTH reply
    1: zbus error: D-Bus handshake failed: Unexpected server AUTH reply
    2: D-Bus handshake failed: Unexpected server AUTH reply
```

**Comment by bstrie** [comment]
@nickvidal What platform are you on?

**Comment by nickvidal** [comment]
> @nickvidal What platform are you on?

Ubuntu 22.04

**Comment by platten** [comment]
Closing in favor of #42 which includes fixes

**Pull request closed without merging**